hack/verify-lint.sh

```diff
@@ -1,65 +0,0 @@
-#!/usr/bin/env bash
-
-# standard bash error handling
-set -o nounset # treat unset variables as an error and exit immediately.
-set -o errexit # exit immediately when a command fails.
-set -E         # needs to be set if we want the ERR trap
-
-readonly CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-readonly ROOT_PATH=$( cd "${CURRENT_DIR}/.." && pwd )
-readonly TMP_DIR=$(mktemp -d)
-readonly GOLANGCI_LINT_VERSION="v1.52.1"
-
-source "${CURRENT_DIR}/utilities.sh" || { echo 'Cannot load CI utilities.'; exit 1; }
-
-cleanup() {
-    rm -rf "${TMP_DIR}" || true
-}
-
-trap cleanup EXIT SIGINT
-
-golangci::install() {
-  export PATH="${INSTALL_DIR}:${PATH}"
-
-  shout "Install the golangci-lint in version ${GOLANGCI_LINT_VERSION}"
-  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b "${INSTALL_DIR}" ${GOLANGCI_LINT_VERSION}
-  echo -e "${GREEN}√ install golangci-lint${NC}"
-
-}
-
-golangci::run_checks() {
-  shout "Run golangci-lint checks"
-  LINTS=(
-    # default golangci-lint lints
-    errcheck gosimple govet ineffassign staticcheck \
-    typecheck unused \
-    # additional lints
-    revive gofmt misspell gochecknoinits unparam exportloopref gosec
-  )
-
-  ENABLE=$(sed 's/ /,/g' <<< "${LINTS[@]}")
-
-  echo "Checks: ${LINTS[*]}"
-  cd ${ROOT_PATH}
-  golangci-lint --disable-all --enable="${ENABLE}" --timeout=10m run --config $CURRENT_DIR/.golangci.yml
```

Comment:
We also need to make sure that we are using the golangci config in the job args

Reply:
but I moved that `.golangci.yml` to root folder already.